packages/office-ui-fabric-react/src/components/Link/__snapshots__/Link.test.tsx.snap

```diff
@@ -15,7 +42,52 @@ exports[`Link renders Link correctly 1`] = `
 exports[`Link renders Link with no href as a button 1`] = `
 <button
   aria-disabled={undefined}
-  className="ms-Link"
+  className=
+      ms-Link
+      {
+        background: none;
```

Comment:
in the snapshot test, can you make sure that we try this with a custom classname, so that we can ensure a customer can do that?

Reply:
Same as above, the component doesn't support a custom `className`. Do we want to add those in?

Reply:
See above. This has been added. :)